sparse/client.go

```diff
@@ -154,7 +154,7 @@ func (client *syncClient) sendHTTPRequest(method string, action string, queries
 	}
 	req.URL.RawQuery = q.Encode()
 
-	log.Debugf("method: %s, url with query string: %s, data len: %d", method, req.URL.String(), len(data))
+	// log.Debugf("method: %s, url with query string: %s, data len: %d", method, req.URL.String(), len(data))
```

Comment:
Can we use trace instead of commenting out? if it's helpful for the development. The same suggestion for others.

Reply:
OK. Let me try it first.

Reply:
Updated